librz/util/compression.c

```diff
@@ -12,8 +12,8 @@
 #include <lzma.h>
 #endif
 
-// set a maximum output buffer of 50MB
-#define MAXOUT 50000000
+// set a maximum output buffer of 5000MB
+#define MAXOUT 5000000000
```

Comment:
this should be configurable.

Reply:
WDYM? We don't store any context for these inflate functions, thus get/set API is not suitable. What I could do is to create some kind of "DecompressOpts" or similar.

Reply:
for now is ok, but these seems to be in place to avoid to alloc too much mem

Reply:
True. But it made me puzzled at first why decompression failed on some debug files and why it didn't. It took a while to realize this comes from this limitation. This is why I also extended error messages.

Reply:
It didn't affect CI in any way. Could we merge this now, and maybe do a proper API change afterwards? Because this blocks many big ELF files with debug information compiled with relatively recent compilers (but before they introduced ZSTD).